Unity-Technologies/ScriptableRenderPipeline
# Added support for stereo camera motion vectors

### Purpose of this PR
I'm using this PR to review my method for updating non-compute shaders. This PR contains changes to enable single-pass stereo rendering of motion vectors, including changes to GetPositionInputs. Please give feedback on these changes, to set the template for future stereo-enabling non-compute-shader changes.

---
### Release Notes
+ Removed jitter from stereo (maybe not necessary?)
+ Added GetPositionInputs_Stereo
+ GetPositionInputs now invokes GetPositionInputs_Stereo with eye = 0
+ Added stereo matrix history
In order to fully enable TAA, need changes to postprocessing:
https://github.com/Unity-Technologies/PostProcessing/tree/pp_vr

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=hdrp_xr_taa&unity_branch=trunk&automation-tools_branch=add-platform-filter

**Manual Tests**: What did you do?
- Used RenderDoc to verify per-eye NDC/WS and to visually inspect motion vectors
- Visually confirmed no additional artifacts with temporal anti-aliasing enabled

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High?
Low

**Halo Effect**: None, Low, Medium, High?
High- GetPositionInputs is used in many places, and these changes will set the template for all future shader changes to enable stereo.

---
### Comments to reviewers
Looking for HDRP-specific feedback from either Sebastien or Julien, and VR coding style feedback from Brandon. (:
